patchsorter/utils/__init__.py

```diff
@@ -36,7 +36,8 @@ def get_torch_device(gpuid = None):
     return device
 
 ################################################################################
-
+def init_worker():
```

Comment:
What functionality does this add?

Reply:
I didn't add this function. I moved this function from **PS.py**, which was **patchsorter/__init__.py**, to **patchsorter/utils/__init__.py**. At **patchsorter/__main__.py** called **init_worker()** so that I have to **import signal** since init_worker() need it.